NDTensors/src/unspec_eltype/unspecified_zero.jl

```diff
@@ -4,6 +4,8 @@
 
 struct UnspecifiedZero <: Base.AbstractFloat end
 
+Base.Complex{UnspecifiedZero}() = complex(UnspecifiedZero())
+Base.Complex{UnspecifiedZero}(z::Real) = (z == zero(eltype(z)) ? complex(UnspecifiedZero()) : throw(ErrorException) )
```

Comment:
Should be able to use `iszero(z)` as a simpler version of `z == zero(eltype(z))`.

Reply:
Thanks, I updated it!